profanity.rb

```diff
@@ -385,7 +385,7 @@ def update(new_value)
 	def redraw
 		setpos(0,0)
 		if @value
-			if @value.class == Fixnum
+			if @value.class == Integer
```

Comment:
I suspect this will probably break for pre-Integer unification versions of Ruby. What do you think about doing something like:

`if @value.is_a?(Integer) || @value.is_a?(Fixnum)` 

instead?

Reply:
I haven't noticed any problems with older ruby and this change but that makes sense.